WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -252,7 +252,7 @@ def _installDIRAC( self ):
     for line in output.split('\n'):
       try:
         var = line.split( '=' )[0].strip()
-        value = line.split( '=' )[1].strip()
+        value = '='.join( line.split( "=" )[1:] ).strip()
```

Comment:
There is the `maxsplit` argument for split https://docs.python.org/2/library/string.html#string.split